UPGRADE.md

```diff
@@ -8,6 +8,16 @@ awareness about deprecated code.
 
 # Upgrade to 3.3
 
+## Deprecated omitting `Comparator::compareSchemas()` third argument
```

Comment:
I don't think it's the right wording. It should be about "If you want, you can pass this argument as of this release until next major". Nothing is being deprecated here.